packages/ui/src/accounts/components/SelectAccount/SelectAccount.tsx

```diff
@@ -23,11 +25,16 @@ interface Props {
   filter?: (option: Account) => boolean
   selected?: Account
   disabled?: boolean
+  minBalance?: BN
 }
 
-export const SelectAccount = React.memo(({ onChange, filter, selected }: Props) => {
+export const SelectAccount = React.memo(({ onChange, filter, selected, minBalance }: Props) => {
```

Comment:
The problem with `minBalance` is that it mixes this logic with filters and might be very limiting. At this stage, I can't be sure that we will filter only by minimal required balance or do we allow a "compatible" lock to be present as well.

Remember, that locked value can be reused for different purposes. So if you have 2 locks and would like to stake (create 3rd lock) that can be set with those 2 we'd need to have a more complex check (filter) for this.

Right now, the simple totalBalance >= minStake is used as it is good enough. We can add a better check later, but we should be prepared to update this check without the need to modify the component itself (something along the lines of open-closed principle).